webpack.config.js

```diff
@@ -1,26 +1,27 @@
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const path = require("path");
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
 module.exports = {
   module: {
     rules: [
       {
         test: /\.css$/,
-        use: ["style-loader", "css-loader"]
+        use: ['style-loader', 'css-loader'],
       },
       {
         test: /\.js$/,
         exclude: /node_modules/,
-        use: ["babel-loader"]
+        use: ['babel-loader'],
       },
       {
         test: /\.(png|svg|jpg|jpeg|gif)$/i,
         type: 'asset/resource',
       },
-    ]
+    ],
   },
```

Comment:
- [ ] Good Job configuring the webpack but there is a small problem. the default mode of your webpack is not set to `development` which results in a warning when we run the server. so kindly consider adding the following line of code at line 22 in webpack.config.js to avoid that warning.
`mode: 'development',`